app/helpers/administrate/application_helper.rb

```diff
@@ -16,5 +16,25 @@ def display_resource_name(resource_name)
           default: resource_name.to_s.pluralize.titleize,
         )
     end
+
+    def svg_tag(asset, options = {})
+      svg_id = options.fetch(:svg_id)
```

Comment:
Since `svg_id` is required, what do you think about making it the second parameter instead of a hash key?